indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskQueue.java

```diff
@@ -228,6 +235,13 @@ public ScheduledExecutors.Signal call()
           }
       );
       requestManagement();
+      // Remove any unacquired locks from storage
```

Comment:
Thanks for adding the comment!
I am a little unclear on why there would be unacquired locks left behind. I would imagine that `shutdown()` would take care of this. If not, please rephrase the comments to clarify that point.

Reply:
shutdown() only clears tasklocks for which a TaskLockPosse is present. The error here is that a lock could not be reacquired and a TaskLockPosse doesn't exist for the conflicting task, which is why these "unacquired" entries must be removed in this manner

Reply:
Okay, please include this info in the comments.

Reply:
Included